index.js

```diff
@@ -10,15 +10,29 @@ const fileRead = async (fileName) => {
   return fileData;
 };
 
+// Function to return the words of the data provided.
+const fileWords = async (data) => {
+  const words = data.split(" ").length;
+  console.log(`${chalk.yellowBright.bold("Number of words: " + words)}`);
+};
+
+// Function to return the sentences of the data provided.
+const fileSentences = async (data) => {
+  const sentences = data.match(/[\w|\)][.?!](\s|$)/g).length;
+  console.log(`${chalk.yellowBright.bold("Number of sentences: " + sentences)}`);
+};
+
 (async () => {
   let [command, filename, delay] = process.argv.slice(2);
-
-  const usage = `${chalk.yellowBright.bold("Command: ")}
+  const usage = `${chalk.yellowBright.bold("Commands: ")}
   ${chalk.green("start")} ${chalk.cyan(
     "<file_path> [<delay>]"
   )} : "Show text from file in <file_path> with delay between words in <delay>" ${chalk.cyan(
     "(default 250WPM)"
   )}
+  ${chalk.green("stats")} ${chalk.cyan(
+    "<file_path>"
+  )} : "Show number of words, sentences, and absolute path of file in <file_path>"
```

Comment:
add suitable color to <file_path>

Reply:
did you want me to change the color of the second <file_path>?

Reply:
Yes

Reply:
the one in this line 
![image](https://user-images.githubusercontent.com/42750440/196299018-7947ba5b-ebb3-4890-9487-82527b4e3ffc.png)

Reply:
perfect